ADALiOS/ADALiOS/ADUserInformation.h

```diff
@@ -38,6 +38,8 @@
 /*! Determines whether userId is displayable */
 @property (readonly) BOOL userIdDisplayable;
 
+@property (readonly) NSString* uniqueId;
```

Comment:
Add comment here. This is passed to ADUserIdentifier as ID when type if UniqueId. This is not displayable so not used as login_hint. Adal.Net stores items in the cache for uniqueID and displayableid.